applications/ConvectionDiffusionApplication/tests/test_convection_diffusion_embedded_solver.py

```diff
@@ -121,18 +158,46 @@ def _addReferenceValuesCheck(self):
     def tearDown(self):
         with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
             KratosUtilities.DeleteFileIfExisting("square.post.bin")
+            KratosUtilities.DeleteFileIfExisting("square_distance_modification.post.bin")
+            KratosUtilities.DeleteFileIfExisting("square_mls_constraints.post.bin")
```

Comment:
Maybe a stupid question but, how would you visualize the output if you're deleting it? :sweat_smile:

Reply:
By putting '#' in front of the files that I want to keep when I set `print_output=true`? :D
I added an if clause now ;)